### ISSUE TITLE
Documented OpenTOL web service API in NAR annual WS issue

### ISSUE THREAD

**Issue by rvosa** [OpenTreeOfLife/hackathon#20]
The journal Nucleic Acids Research has an annual issue dedicated to web services. The deadline appears to be around Christmas. It would make sense if one of the deliverables of the hackathon would be an entry in this issue. As submissions must include easy-to-use client code templates there is scope for non-OpenTOL people to contribute to this. http://www.oxfordjournals.org/our_journals/nar/for_authors/submission_webserver.html

**Comment by arlin** [comment]
Yes!  This also gives OpenTree something very specific to aim for in terms of ensuring that its services are running in 1 year when the annual DB issue is released.

**Comment by kcranston** [comment]
Great idea! I didn't know about this. Even if it didn't happen at the hackathon, something we should do.

**Comment by rvosa** [comment]
Just to clear up the OP a bit: I don't think a hackathon is the right place to do the actual writing (and I don't think that would be all that much work anyway), but it would be good to keep it in the back of our minds and make sure all the other bits (correct licensing, documentation, client code, test cases) are in place in order to write the vignette.
